decentraland/screens/BuyLandScreen.js

```diff
@@ -53,16 +60,29 @@ export class BuyLandScreen extends React.Component {
 
     const onSuccess = () => {
       showInfo(`${typeDescription} bought successfully.`);
-      removeLandForSale(landForSale);
-      addToMyAssetsList(asset);
+    };
+
+    const onError = () => {
+      removeMyAssetFromList(asset);
```

Comment:
We probably want to show an alert or toast in this case

Reply:
Yes.

Reply:
Great, thanks

Reply:
Done.

Reply:
Resolving this. Thanks.